src/hooks/useJobProfiles.js

```diff
@@ -0,0 +1,31 @@
+/* istanbul ignore file */
+import { useQuery } from 'react-query';
+import { useNamespace, useOkapiKy } from '@folio/stripes/core';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
It is worth changing this in eslint config to avoid having to keep doing this.

Reply:
🤷 Or just `import useWhatever from '../path/to/hook'` instead of `import { useWhatever } ..." It is unfortunate that the convention for exporting components (default export) is the opposite of the convention for hooks (named export). In any case, @Dmytro-Melnyshyn , open a Jira against ESCONF and a corresponding PR in eslint-config-stripes to set the "prefer default export" option to false.

Reply:
I've converted it default at least untill we have this eslint rule to be consistent.